src/Reflection/ReflectionClass.php

```diff
@@ -71,7 +70,7 @@ private function __construct()
      * @param string $className
      * @return string
      */
-    public static function export($className = null)
+    public static function export(string $className = null) : string
```

Comment:
Both of course!
![becausephp](https://cloud.githubusercontent.com/assets/144181/24584371/eddde0ba-176b-11e7-90ca-77a41f193702.jpg)